.gitignore

```diff
@@ -1 +1,2 @@
 /node_modules/
+dist/
```

Comment:
We should also add a [`.eslintignore`](https://github.com/bpmn-io/moddle-xml/blob/c38e5b2932f77174e2263a6eaaba328a16b3c75d/.eslintignore) file here:

```json
dist/
```

Without `npm run lint` would also check the bundles.